### PR TITLE
Add metrics for sli/slo

### PR DESCRIPTION
**Pull request by jjcomer**
We need to be able to count the number of repo pulls that take too long and those that time out. In datadog we are only able to express this as a ratio of count metrics and cannot use the existing timer metric.

Also updated deps

### CODE REVIEW

File: Cargo.toml [hunk 1]
```diff
@@ -16,23 +16,23 @@ edition = '2018'
...
-stderrlog = '0.4'
+stderrlog = '0.5'
 structopt = '0.3'
 tempfile = '3'
 url = '2'
 walkdir = '2'
 zip = '0.5'
 dogstatsd = '0.6'
-actix-web = '2'
+actix-web = '3.1'
```
Comment (jonathanmorley) [inline Cargo.toml:33]: how big of a bump is this, should we be concerned about breakages?

File: src/app/fetch_actor.rs [hunk 2]
```diff
@@ -41,23 +41,41 @@ impl Receive<ExecuteFetch> for FetchActor {
...
+                debug!(
+                    "Performed scheduled repo fetch took: {} ms. Poll delay: {} ms",
+                    elapsed_time.as_millis(),
+                    self.last_updated
+                        .elapsed()
+                        .unwrap_or_else(|_| Duration::from_secs(0))
+                        .as_millis()
+                );
+
+                //Only record time of successful fetches
```
Comment (jonathanmorley) [inline src/app/fetch_actor.rs:62]: are we recording failure to fetch anywhere? even if we don't record the time

Comment (jjcomer) [reply src/app/fetch_actor.rs:62]: A few lines below we count the failure

File: Cargo.toml (hunk 1 above)
Comment (jjcomer) [reply Cargo.toml:33]: The major is due to some module renames

**Review by jonathanmorley — APPROVED** [review]

**Pull request merged**